### PR TITLE
Added container image.

### PR DESCRIPTION
**Pull request by HU90m**
Added a container image for running simulations.

### CODE REVIEW

File: container/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,96 @@
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+
+# Docker container containing various hardware and software development tools
+# for OpenTitan.
```
Comment (ctopal) [inline container/Dockerfile:6]: We might want to change this to say `for Demo System`

**Review by ctopal — COMMENTED** [review]
I think a mention for the docker in the README of the repo is needed with this PR. Also a mention to install docker might be good for clueless people like me :)

Otherwise I tested it and it works.

**Comment by hcallahan-lowrisc** [comment]
> I think a mention for the docker in the README of the repo is needed with this PR. Also a mention to install docker might be good for clueless people like me :)
> 
> Otherwise I tested it and it works.

Yes, Docker/Podman installer instructions / link would be super valuable.

**Review by hcallahan-lowrisc — APPROVED** [review]
I think this is good to go in, with a separate task to do the docs stuff.

**Pull request merged**